.github/workflows/php.yaml

```diff
@@ -0,0 +1,30 @@
+name: PHP
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron:  '0 3 * * *'
+
+jobs:
+  analyze:
+    name: "Code style and static analysis"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v1
+        with:
+          php-version: '7.4'
+          extensions: mbstring, intl, zip
+
+      - name: Install dependencies
```

Comment:
you could add a cache section to speed up composer ?

Reply:
:+1: True, we may improve it in the future, but first lets get Actions working at least somehow :)